### PR TITLE
A set of updates - Set MailHog settings

### PR DESCRIPTION
**Pull request by GrCOTE7**

### CODE REVIEW

**Comment by GrCOTE7** [comment]
Hello, **BestMomo**...

Not at all! These are just additional tools we'll need in the future.

Currently, everything is placed in resources/views/livewire/gc7/frameworks... etc.

However, upon reflection, all of this will be part of a section not named GC7, but rather "student".
In fact, we can have a project that includes the real future website, as well as everything that helps to learn or remember various functionalities!

If the user isn't logged in:
https://prnt.sc/_p-k60qcOgaI

Or if logged in as 'admin' ([admin@example.com](mailto:admin@example.com)):
https://prnt.sc/_sEcsMXZatPl

Then everything appears 'normal', seemingly exactly as you've done it!

However, if they log in as 'student' ([student@example.com](mailto:student@example.com)), globally in the blog sillo.laravel.com, everything is standard (So as soon as continuous integration is set up, a simple push of the master branch will update the real future website - Note: This setup will only take about 5 minutes, just once!). But a new menu item appears:

Currently: **Test | Frameworks**:
https://prnt.sc/IdTOnFrGhPyK

This gives them access to numerous scripts to help them, etc., or spaces to test anything they want.

Initially, this was 'just for me'... But after reflection, all future contributors will find it interesting and useful.

In summary, it's complementary but fully aligns with the desire to help the French-speaking community overcome all the subtleties of Laravel!

Note, and remember: That's only a suggestion, Captain ;-) !
